src/mobile/components/layout/test/layout.test.coffee

```diff
@@ -189,3 +189,31 @@ xdescribe 'afterSignUpAction', ->
     )
     @bootstrap()
     @Cookies.expire.args[0][0].should.equal 'afterSignUpAction'
+
+describe 'logoutEventHandler', ->
```

Comment:
I only wrote tests for `logoutEventHandler` (and not for the listener setup) because there's apparently an unresolved issue with testing `bootstrap` that's preventing all of the tests in this file from being run (https://github.com/artsy/force/blob/fa72fb7004a9b59d226a723a45db574e64ad7638/src/mobile/components/layout/test/layout.test.coffee#L14)